MAUI/PDF-Viewer/Shapes.md

```diff
@@ -19,14 +19,16 @@ The following shape annotation types are now available in the PDF Viewer.
 2.	Circle.
 3.	Line.
 4.	Square.
+5.	Polygon.
+6.	Polyline.
 
 ## Add shapes to a PDF document
 
 This section will go through how to add shape annotations to a PDF document interactively by drawing on the PDF document as well as programmatically.
 
 ### Add shapes with UI interaction
 
-You can draw and add shape annotations to a PDF document interactively by touch (or mouse down) and drag. The following steps explains how to draw shape annotation on a PDF.
+You can draw and add shape annotations to a PDF document interactively by touch (or mouse down) and drag. The following steps explains how to draw circle shape annotation on a PDF. Similarly, you can draw other shapes like Arrow, Line and Rectangle.  
```

Comment:
You can interactively draw and add shape annotations to a PDF document by touching or dragging the mouse down. The following steps explain how to draw a circle-shaped annotation on a PDF. Similarly, you can draw other shapes like arrows, lines, and rectangles.